src/js/components/PlacementConstraintsPartial.js

```diff
@@ -136,8 +107,17 @@ export default class PlacementSection extends Component {
               value={String(constraint.operator)}
             >
               <option value="">Select</option>
-              {this.getOperatorTypes()}
+              {Object.keys(OperatorTypes).map((type, index) => {
+                return (
+                  <option key={index} value={type}>
+                    {type}
+                  </option>
+                );
+              })}
             </FieldSelect>
+            <FieldHelp>
+              Specify where your app will run
```

Comment:
Please add a period here, sir. Or get rid of the period on "A string, integer or regex value."